src/actions/stories.action.js

```diff
@@ -0,0 +1,66 @@
+import * as actionTypes from './actionTypes'
+import Api from './../utils/apiWrapper'
+
+function storiesLoaded(stories) {
+  return { type: actionTypes.STORIES_LOADED, payload: stories }
+}
+
+function storyCreated(storyId, storyName) {
+  return {
+    type: actionTypes.STORY_CREATED,
+    payload: { id: storyId, name: storyName }
+  }
+}
+
+function storyEdited(storyId, storyName) {
+  return {
+    type: actionTypes.STORY_EDITED,
+    payload: { id: storyId, name: storyName }
+  }
+}
+
+function storyDeleted(storyId) {
+  return { type: actionTypes.STORY_DELETED, payload: { id: storyId } }
+}
+
+export function getStories() {
+  return dispatch => {
+    return Api.getStories().then(story => dispatch(storiesLoaded(story)))
+  }
+}
+
+export function getStoriesByPOI(poiId) {
+  return dispatch => {
+    return Api.getStoriesByPOI(poiId).then(storyIds =>
+      dispatch(storiesLoaded(storyIds))
+    )
+  }
+}
+
+export function getStory(storyId) {

```

Comment:
this endpoint won't exist in api v2 so you can safely get rid of it (make sure to get rid of its imports and exports too)